.github/workflows/integration-tests-on-release.yaml

```diff
@@ -135,7 +135,7 @@ jobs:
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
     - name: setup tmate session for debugging
-      if: failure() && contains(github.event.pull_request.labels.*.name, 'debug-helm-suite')
```

Comment:
Should we only enable this one after the helm tests are fixed in master?

Reply:
We need this to actually debug helm further.

Reply:
Ok, will give it a try troubleshooting today with this!